featurebyte/common/documentation/documentation_layout.py

```diff
@@ -852,6 +852,7 @@ def _get_utility_classes_layout() -> List[DocLayoutItem]:
             doc_path_override="api.groupby.GroupBy.forward_aggregate_asat.md",
         ),
         DocLayoutItem([UTILITY_CLASSES, FEATURE, "FeatureVersionInfo"]),
+        DocLayoutItem([UTILITY_CLASSES, WAREHOUSE, "BigQueryDetails"]),
```

Comment:
If we don't expose this in the documentation and don't expose the new class in the top level init file, would that allow us to merge changes to main branch incrementally? I think that can simplify the development process.